### PR TITLE
Solution using PowerShell with embedded C#

### PR DESCRIPTION
**Pull request by RobCannon**
## Description
This is a PowerShell implementation of davepl's sieve of Erastosthenes for
benchmarking programming languages and CPUs. This implementation uses the
capability of PowerShell to build a dynamic assembly from C# and run that.
The C# code is based on the current fastest C# implementation by tannergooding,
solution_3. It has long been a capability of PowerShell to allow a script to access 
.NET directly when it is needed for speed or accessing some feature not available 
in PowerShell. This capability is built into PowerShell and does not need any 
external dependencies.  This is considered a mixed solution for the drag-race
and is thus not considered faithful.

## Contributing requirements
* [x] I read the contribution guidelines in CONTRIBUTING.md.
* [x] I placed my solution in the correct solution folder.
* [x] I added a README.md with the right badge(s).
* [x] I added a Dockerfile that builds and runs my solution.
* [x] I selected `drag-race` as the target branch.
* [x] All code herein is licensed compatible with BSD-3.

### CODE REVIEW

**Comment by rbergen** [comment]
@RobCannon Could you please sync your fork with this branch? It should enable CI for your PR.

**Comment by RobCannon** [comment]
Done

File: PrimeMixed/solution_2/README.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+## Output
+Intel Core i7-1165G7 @ 2.80GHz on Windows 11 with PowerShell 7.1.3 in Windows Subsystem for Linus for a sieve size of 10, 100, 1000, 10000, 100000 and 1000000:
+
+```
+RobCannon_ps4;29621618;5.00004;1;algorithm=base,faithful=yes,bits=1
+RobCannon_ps4;23481503;5.00089;1;algorithm=base,faithful=yes,bits=1
+RobCannon_ps4;6305039;5;1;algorithm=base,faithful=yes,bits=1
+RobCannon_ps4;704742;5;1;algorithm=base,faithful=yes,bits=1
+RobCannon_ps4;66692;5.00007;1;algorithm=base,faithful=yes,bits=1
+RobCannon_ps4;5984;5.00021;1;algorithm=base,faithful=yes,bits=1
```
Comment (rbergen) [inline PrimeMixed/solution_2/README.md]: The output lines in the README still label the implementations as faithful. I'm not going to withhold approval over this, but you may want to change this nonetheless.

Comment (RobCannon) [reply PrimeMixed/solution_2/README.md]: Oops, missed that.  That is fixed now.

**Review by rbergen — APPROVED** [review]
LGTM

**Review by marghidanu — APPROVED** [review]

**Pull request merged**